x/pki/types/message_add_noc_x_509_ica_cert.go

```diff
@@ -13,12 +13,11 @@ const TypeMsgAddNocX509IcaCert = "add_noc_x_509_ica_cert"
 
 var _ sdk.Msg = &MsgAddNocX509IcaCert{}
 
-func NewMsgAddNocX509IcaCert(signer string, cert string, certSchemaVersion, schemaVersion uint32) *MsgAddNocX509IcaCert {
+func NewMsgAddNocX509IcaCert(signer string, cert string, schemaVersion uint32) *MsgAddNocX509IcaCert {
```

Comment:
It might be better to use the `certSchemaVersion` instead of `schemaVersion` to be consistent with the field name `CertSchemaVersion` in `MsgAddNocX509IcaCert`